js/maps-setup.js

```diff
@@ -31,6 +32,11 @@ var myGeoJSON= {
                 "coordinates":[[[-85.60546875,49.03786794532644],[-96.6796875,40.713955826286046],
                                 [-79.62890625,37.71859032558816],[-81.2109375,49.26780455063753],
                                 [-85.60546875,49.03786794532644]]]}},
```

Comment:
These are *really* far way from london... where did you type in the co-ords? I don't see them here in the code.